include/common_structs.h

```diff
@@ -18,7 +24,7 @@ typedef struct cam_pos_settings {
     /* 0x04 */ f32 boomLength;
     /* 0x08 */ f32 boomPitch;
     /* 0x0C */ f32 viewPitch;
-    /* 0x10 */ f32 position[3];
+    /* 0x10 */ struct vec3f position;
```

Comment:
if we do end up using Vec3f, it needs to be changed here. Also, you can leave the struct off, and just use Vec3f.

Reply:
I use `struct` everywhere in this file because it's what the `starrod_structs_to_h.py` generates, but we can (should?) change that.

Reply:
I tried changing to avoid `struct` where possible, but it forces us to declare types before they are used meaning reordering everything in the file with zero-dependency structs at the top. I reckon we should leave `struct` there until we split `common_structs.h` into a bunch of header files closer to the c files which use them.